static/app/views/settings/organizationTeams/teamMembers.tsx

```diff
@@ -178,7 +154,7 @@ class TeamMembers extends Component<Props, State> {
           this.setState({
             loading: false,
             error: false,
-            teamMemberList: this.state.teamMemberList.concat([orgMember]),
+            teamMemberList: this.state.teamMemberList.concat([orgMember as TeamMember]),
```

Comment:
How does orgMember differ here?
Can we modify orgMember to satisfy the teamMember type rather than changing the orgMembers type?

Reply:
I've intentionally done this to avoid polluting `Member` type with once-off keys that's specific to this endpoint. I can't make a better design without breaking compatibility. 

Ideally for TeamMembers, I'd want to return the Team object with a list of all its members.